cypress.json

```
@@ -9,37 +9,6 @@
 	"experimentalFetchPolyfill": true,
 	"env": {
 		"DEBUG": "false",
-		"INCLUDED_V2_TEST_FILES": [
-			"accordion",
-			"blockquote",
-			"blog-posts",
-			"button",
-			"card",
-			"column",
-			"columns",
-			"container",
-			"count-up",
-			"cta",
-			"divider",
-			"expand",
-			"feature",
-			"feature-grid",
-			"header",
-			"heading",
-			"icon",
-			"icon-list",
-			"image-box",
-			"notification",
-			"number-box",
-			"pricing-box",
-			"separator",
-			"spacer",
-			"team-member",
-			"testimonial",
-			"text",
-			"video-popup"
-		],
-		"INCLUDED_V3_TEST_FILES": [],
 		"FORCE_INCLUDE_ALL_TESTS": "true",
```

```suggestion
		"FORCE_INCLUDE_ALL_TESTS": "true",
```

remove this one as well